test/utils/parsers/get_applications_parser.go

```diff
@@ -19,19 +19,22 @@ func ParseJxGetApplications(s string) (map[string]Application, error) {
 	answer := make(map[string]Application, 0)
 	lines := strings.Split(strings.TrimSpace(s), "\n")
 	headerFound := false
+	fieldCount := 3
 	for _, line := range lines {
 		// Ignore any output before the header
 		if strings.HasPrefix(line, "APPLICATION") {
 			headerFound = true
+			headers := strings.Fields(strings.TrimSpace(line))
+			fieldCount = len(headers)
 			continue
 		}
 		if !headerFound {
 			continue
 		}
 		line = strings.TrimSpace(line)
 		fields := strings.Fields(line)
-		if len(fields) < 3 {
-			return nil, errors.Errorf("must be at least 3 fields in %s, entire output was %s", line, s)
+		if len(fields) < fieldCount {
```

Comment:
should it not be 4? four is what's later triggers the split. Not sure whether we have to look at the function as a whole here.

Reply:
Might be worth to do something like the following as well:

```
urlString := fields[len(fields)-1]
u, err := url.Parse(urlString)
if err != nil {
   return err
}
```